src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.container {
+  margin: 20px 0;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 700;
+  color: #3d4e61;
+  font-family: avenir, sans-serif;
+  background: url(images/Search.svg) no-repeat ;
+}
+
+.big__search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-position: left 26px bottom 25px ;
+}
+
+.small__search {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: left 12px bottom 15px;
+}
+
+.big__search:focus {
+  background: url(images/Search.svg) no-repeat left 26px bottom 25px ,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.small__search:focus {
+  background: url(images/Search.svg) no-repeat left 12px bottom 15px,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-size: 11px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.search::placeholder {
+  font-family: avenir, sans-serif;
+  color: #3d4e61;
+  font-weight: 400;
+}
```

Comment:
write this style after search class selector (all styles for one element together)